src/request/import/ImportWords.js

```diff
@@ -130,10 +131,14 @@ class ImportWords {
             window.location.hash = ImportWords.Pages.DOWNLOAD_LOGINFILE;
         });
 
-        $skipDownloadButton.addEventListener('click', e => {
-            e.preventDefault();
-            this._resolve(this._keyResults);
-        });
+        if (this._request.wordsOnly && this._request.requestedKeyPaths) {
+            $skipDownloadButton.remove();
```

Comment:
Is this the correct condition? What do the requested key paths have to do with removing the skipDownloadButton? Do you mean to check for a given `expectedKeyId` instead?